src/functional.test.ts

```diff
@@ -289,5 +272,289 @@ describe('create-release-branch (functional)', () => {
+          expect(
+            await environment.readJsonFileWithinPackage('c', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/c',
+            version: '2.0.13',
+          });
+          expect(
+            await environment.readJsonFileWithinPackage('d', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/d',
+            version: '1.2.3',
+          });
+        },
+      );
+    });
+
+    it('does not update the versions of any packages that have been tagged with intentionally-skip', async () => {
+      await withMonorepoProjectEnvironment(
+        {
+          packages: {
+            $root$: {
+              name: '@scope/monorepo',
+              version: '1.0.0',
+              directoryPath: '.',
+            },
+            a: {
+              name: '@scope/a',
+              version: '0.1.2',
+              directoryPath: 'packages/a',
+            },
+            b: {
+              name: '@scope/b',
+              version: '1.1.4',
+              directoryPath: 'packages/b',
+            },
+            c: {
+              name: '@scope/c',
+              version: '2.0.13',
+              directoryPath: 'packages/c',
+            },
+            d: {
+              name: '@scope/d',
+              version: '1.2.3',
+              directoryPath: 'packages/d',
+            },
+          },
+          workspaces: {
+            '.': ['packages/*'],
+          },
+        },
+        async (environment) => {
+          await environment.runTool({
+            releaseSpecification: {
+              packages: {
+                a: 'major',
+                b: 'intentionally-skip',
+                c: 'patch',
+                d: 'intentionally-skip',
+              },
+            },
+          });
+
+          expect(await environment.readJsonFile('package.json')).toStrictEqual({
+            name: '@scope/monorepo',
+            version: '2.0.0',
+            private: true,
+            workspaces: ['packages/*'],
+          });
+          expect(
+            await environment.readJsonFileWithinPackage('a', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/a',
+            version: '1.0.0',
+          });
+          expect(
+            await environment.readJsonFileWithinPackage('b', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/b',
+            version: '1.1.4',
+          });
+          expect(
+            await environment.readJsonFileWithinPackage('c', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/c',
+            version: '2.0.14',
+          });
+          expect(
+            await environment.readJsonFileWithinPackage('d', 'package.json'),
+          ).toStrictEqual({
+            name: '@scope/d',
+            version: '1.2.3',
+          });
+        },
+      );
+    });
+
+    it('does not update the changelogs of any packages that have been tagged with --intentionally-skip', async () => {
```

Comment:
Nit: I spent a while comparing this with the previous test, trying to see what the difference between `intentionally-skip` and `--intentionally-skip` was. I'm guessing it was a typo? This normalizes the two test descriptions

```suggestion
    it('does not update the changelogs of any packages that have been tagged with intentionally-skip', async () => {
```

Reply:
Yes, thank you!